lesson03/assignment/index.js

```diff
@@ -0,0 +1,36 @@
+const cardsJSON = '[{"name": "Paolo Maldini","jobTitle": "Front End Developer","company": "Web Weavers","experience": "3 years","school": "University of Washington","major": "Marketing","email": "paolo@example.com","linkedInUrl": "paolo.linkedinprofile.com","codeLanguages": ["HTML", "CSS", "JavaScript", "Node", "Express"]},{"name": "Barbara Bonansea","jobTitle": "Software Engineer","company": "Excellence in the Cloud","experience": "12 years","school": "University of Southern California","major": "Computer Science","email": "barbara@example.com","linkedInUrl": "barbara.linkedinprofile.com","codeLanguages": ["HTML", "JavaScript", "C", "Go"]},{"name": "Javier Hernandez","jobTitle": "User Experience Engineer","company": "Web Sites and More","experience": "5 years","school": "Seattle University","major": "Performing Arts","email": "javier@example.com","linkedInUrl": "javier.linkedinprofile.com","codeLanguages": ["HTML", "CSS"]},{"name": "Maribel Dominguez","jobTitle": "Front End Engineer","company": "Bits and Bytes","experience": "6 years","school": "University of Washington","major": "Mechanical Engineering","email": "maribel@example.com","linkedInUrl": "maribel.linkedinprofile.com","codeLanguages": ["HTML", "CSS", "JavaScript", "React", "Vue", "Redux"]}]';
+
+// turning the JSON into a javascript object
+const cards = JSON.parse(cardsJSON);
+// container for the template-hook section
+const container = document.querySelector('.template-hook');
+
+// TO DO: Make output variable hold html with info about each person card in the list
+//    a. in class card, input: image, h1 (name), and p (position/jobTitle)
+// Loop through all elements of the list
+cards.forEach(function(c){
+  // Build HTML
+  let list = document.createElement('div');
```

Comment:
I would suggest wrapping your `card` and `card-more` into a parent div tag. You will also want to add some CSS for this parent div, adding some margin so the other cards have space between them